.travis.yml

```diff
@@ -4,15 +4,32 @@ python:
   - 3.5
   - 3.6
 
+stages:
+  - test
+  - moban
+
+.disable_global: &disable_global
+  before_install: false
+  install: false
+  script: false
+
+.check_moban: &check_moban
+  <<: *disable_global
+  python: 3.6
+  stage: moban
+  install: pip install moban
+  script: .ci/check_moban.sh
+
+jobs:
+  include:
+    - *check_moban
+
```

Comment:
```
  allow_failures:
    - *check_moban
```
?

Reply:
No ;-)

That is only done on coala & coala-bears as we have lots of eyes on those two repos, and those two repos see enhancements first (like pytest-reqs is only working on coala core repo atm)